pennylane_cirq/qsim_device.py

```diff
@@ -47,14 +47,17 @@ class QSimDevice(SimulatorDevice):
         qubits (List[cirq.Qubit]): A list of Cirq qubits that are used
             as wires. The wire number corresponds to the index in the list.
             By default, an array of ``cirq.LineQubit`` instances is created.
+        qsim_options: (Dict[str, Any]): A dictionary with options for the qsimh simulator. See the `qsim
+            usage documentation <https://github.com/quantumlib/qsim/blob/master/docs/usage.md>`__
+            for further details.
     """
```

Comment:
This doesn't pertain to a version bump, but I think it would be nice to still include it in the 0.14 release.

Reply:
:+1: